### PR TITLE
Backend-Setup

### PR DESCRIPTION
**Pull request by TheShicksinator**
Barebones backend setup is good, accounts and further testing pending.

### CODE REVIEW

File: services/meals.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const { ObjectId } = require("mongodb");
+
+export const createMeal = async (
+    title: string,
+    price: number,
+    description: string,
+    filters: object,
+    postId: string
+) => {
+    const mealCollection = await meals();
```
Comment (A-very-Cunning-ham) [inline services/meals.ts:12]: do we need to create a const for these collections every time even if we just use them once per function?

Comment (TheShicksinator) [reply services/meals.ts:12]: They're block scoped and even more so do to the export so it doesn't really make a difference either way, I figure it's just better practice cause we don't want to accidentally change them.

**Comment by TheShicksinator** [comment]
Routes for API are complete, if there are any additional DB functions y'all think we need, let me know and I'll add them.

**Review by Jordan883 — APPROVED** [review]
Looked it over. I'm untrained in backend coding, but this looks good to me! We can go back to refactor/fix later if necessary. For now, let's get this on the ground so other branches can rebase.

**Pull request merged**